cip-constitution-tech/README.md

```diff
@@ -0,0 +1,255 @@
+
+### File Type
+
+The constitution file MUST be a text file named `constitution.txt`.
+
+### Hashing
+
+When supplying a constitution hash digest to chain, the algorithm used MUST be Blake2b-256.
+Before creating a hash digest the constitution plain text MUST be in its raw text, including any [Rich Text Formatting](#rich-text-formatting) related characters.
+
+### Storage
+
+The each ratified constitution MUST be stored, immutably on a distributed storage mechanism such as IPFS.
+Where backups can be easily made in a permissionless manner by interested parties.
+This storage platform MUST be easily accessible, with strong tooling support.
+Authors SHOULD NOT specify the use of centralized gateways to access the constitution plain text.
+
+// only ascii
+
+### Rich Text Formatting
+
+The constitution text MAY include a strict subset of rich text styling as defined in this specification.
+Tooling rendering the constitution SHOULD recognize these and render them faithfully.
+
+#### Headers
+
+Headers are denoted via a hashtag character `#` followed by a space ` `.
+There are six levels to headers with, each being defined via a extra `#`.
+Headers are ended via a line break.
+Headers SHOULD be followed below by a blank line.
+Headers SHOULD not be preceded by whitespace.
+
+The lower the number of `#` the larger order the text SHOULD be rendered.
+
+Example:
+```md
+# H1
+
+## H2
+
+### H3
+
+#### H4
+
+##### H5
+
+###### H6
+
+```
+
+If text is in a header no other formatting can be applied.
+
+#### Emphasis
+
+Emphasis is applied to text between single or double asterisks, without space between asterisks and text.
+Italicized emphasis is shown via single asterisk (`*`).
+Bold emphasis is shown via double asterisks (`**`).
+
+Emphasis cannot span multiple lines.
+
+Examples:
+```md
+Emphasis, aka italics, with single *asterisks*.
+
+Strong emphasis, aka bold, with double **asterisks**.
+```
+
+The text contained within headings can be emphasized.
+
+Both italicized and bold cannot be applied to the same text.
+
+#### Code and Syntax Highlighting
+
+Texted can be highlighted as code, when encased without spaces by backticks.
+This MUST not contain line breaks.
+
+Example:
+```md
+Inline `code` has `back-ticks around` it.
+```
+
+The text contained within headings or emphasis cannot be highlighted as code.
+
+
+#### Unordered Lists
+
+// todo
+
+#### Line Breaks / Paragraphs
+
+To create paragraphs, use a blank line to separate one or more lines of text.
+
+Examples:
+```md
+Here's a line for us to start with.
+
+This line is separated from the one above by two newlines, so it will be a *separate paragraph*.
+
+This line is also a separate paragraph, but...
+This line is only separated by a single newline, so it's a separate line in the *same paragraph*.
+```
+
+### Best Practices
+- when hash doesnt match
+- when unrecognised formatting/style
+- Keep lines left-aligned without unneeded whitespace or tabs
+- tabs vs spaces
```

Comment:
Not editing this section because it looks like a draft (@Ryun1 please let us know if otherwise)

Reply:
Yup, still a draft
just going to write out some general good practices/advice